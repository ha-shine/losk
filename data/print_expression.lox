// binary and grouping expressions, with precedence
print (1 + 2) * 5 + 2;

print "hello " + "world";

// Logical expressions
print false or true;

// Unary expressions
print -10.5;
print !true;
