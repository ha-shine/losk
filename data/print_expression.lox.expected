17
hello world
true
-10.5
false
